include/hw_constants_checks.h

```diff
@@ -17,11 +16,11 @@ namespace gbt{
      *  \param[in] phase Phase value to check.
      *  \return Returns `false` in case of success; `true` in case of error. The precise error is logged and written to the `error` RPC key.
      */
-    inline bool checkPhase(wisc::RPCMsg *response, uint8_t phase){
-        if (phase < gbt::PHASE_MIN)
-            EMIT_RPC_ERROR(response, stdsprintf("The phase parameter supplied (%hhu) is smaller than the minimal phase (%hhu).", phase, gbt::PHASE_MIN), true)
-        if (phase > gbt::PHASE_MAX)
-            EMIT_RPC_ERROR(response, stdsprintf("The phase parameter supplied (%hhu) is bigger than the maximal phase (%hhu).", phase, gbt::PHASE_MAX), true)
+    inline bool checkPhase(uint8_t phase){
```

Comment:
This should probably not be using `EMIT_RPC_ERROR`, but i don't have a feel if this function should return a `bool`, and have the calling code handle the logic, or raise an exception, and expect the calling code to handle the exception.

Reply:
Now that we can safely use exceptions within the `ctp7_modules`, I'm in favor of raising an exception in such cases. The calling code on the DAQ machine, which gave bad parameters, should be handling the exceptions.